src/ray/gcs/gcs_server/test/gcs_server_rpc_test.cc

```diff
@@ -457,23 +457,6 @@ TEST_F(GcsServerTest, TestActorInfo) {
   JobID job_id = JobID::FromInt(1);
   auto actor_table_data = Mocker::GenActorTableData(job_id);
 
-  // Register actor
```

Comment:
I will remove unused AsyncRegister&AsyncUpdate api in next PR. https://github.com/ray-project/ray/pull/11099